modules/vaos/app/services/vaos/messages_service.rb

```diff
@@ -19,6 +19,7 @@ def post_message(request_id, request_object_body)
         response = perform(:post, messages_url(request_id), params, headers)
 
         if response.status == 200
+          response.body.delete(:sender_id)
```

Comment:
Is this ok syntax? I'm not sure exactly how you note if blocks in Ruby.

Reply:
Yep that's ok. Rails (via the active_support gem) adds an `except` method to Ruby Hashes, so you could also do `response.body.except(:sender_id)`.  Realize the if/else logic was there before, but that could also use a guard clause to handle the non 200 case upfront:

```ruby
handle_error(response) unless response.status == 200

{
  data: OpenStruct.new(response.body.except(:sender_id)),
  meta: {}
}
```

or `if response != 200` if you're allergic to `unless` 😄